BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -333,6 +336,12 @@ public final class BlueprintView: UIView {
 
     public override func didMoveToWindow() {
         super.didMoveToWindow()
+        if supportsAccessibilityLargeContentViewer {
```

Comment:
Should we also check if the window is nil here? this method gets called when we remove a view from the hierarchy.

https://developer.apple.com/documentation/uikit/uiview/didmovetowindow()